output_template/python/lmnet/data_augmentor.py

```diff
@@ -1 +0,0 @@
-../../../lmnet/lmnet/data_augmentor.py
```

Comment:
Need to create `output_template/python/blueoil/data_augmentor.py` ?

Reply:
@suttang 
In `output_template`, We don't use `data_augmentor`.

Reply:
I don't know why anyone create this symbolic link.

Reply:
OH! TASHIKANI!